lab_1_classify_by_unigrams/main.py

```diff
@@ -85,6 +132,35 @@ def compare_profiles(
     'freq' in arguments, None is returned
     """
 
+    if (not isinstance(unknown_profile, dict)
+            or not isinstance(profile_to_compare, dict)):
+        return None
+    if ("name" not in unknown_profile
+            or "freq" not in unknown_profile
+            or "name" not in profile_to_compare
+            or "freq" not in profile_to_compare):
+        return None
+
+    unknown_freq, compare_freq = {}, {}
+    if isinstance(unknown_profile, dict):
+        unknown_freq = unknown_profile["freq"]
+    if isinstance(profile_to_compare, dict):
+        compare_freq = profile_to_compare["freq"]
+    unknown_keys, compare_keys = list(unknown_freq.keys()), list(compare_freq.keys())
+    all_keys = (unknown_keys + list(set(compare_keys) - set(unknown_keys)))
+    all_unknown_profile = dict.fromkeys(all_keys, 0)
+    all_profile_to_compare = dict.fromkeys(all_keys, 0)
+    for (key, value) in unknown_freq.items():
```

Comment:
can you rename key and value for cleaar understanding of meaning of these variables?

Reply:
those correspond to the key and value in the dictionary items respectably
so variable 'key' holds the key of the dictionary item, and 'value' holds value
I'm not sure it can be clearer, to be honest